src/libs/workspace-utils.ts

```diff
@@ -29,6 +29,11 @@ interface BaseWorkspaceInfo {
   createdDate: string;
   createdBy: string;
   lastModified: string;
+  bucketName?: string;
+  attributes?: Record<string, unknown>;
+  isLocked?: boolean;
+  canShare?: boolean;
```

Comment:
I believe `canShare` goes on the outer `BaseWorkspace` type, not the inner `BaseWorkspaceInfo` type.

Reply:
Right again.

I think I ended up with that there when trying to get some js-derived types to work, where I should have just used `@ts-expect-error`.